admin/client.py

```diff
@@ -110,15 +107,28 @@ def make_script_file(dir, effects):
     return os.path.basename(filename)
 
 
-class DockerRunner:
+class DockerContainer:
     """
     Run commands in a Docker container.
     """
 
     def __init__(self, image):
-        self.docker = docker.Client(version='1.18')
+        # See FLOC-3044 - these settings are to avoid problems on OSX
```

Comment:
It isn't just for Boot2Docker and I'm not sure about Homebrew Python, get rid of everything after "[sic] OSX".